frontend/src/framework/store/store.package.ts

```diff
@@ -65,7 +68,9 @@ class Store {
                     thunk: {
                         extraArgument: this.extraArguments,
                     },
-                }).prepend(errorMiddleware),
+                })
+                    .prepend(errorMiddleware)
+                    .concat(chatSocketMiddleware), // eslint-disable-line unicorn/prefer-spread
```

Comment:
?

Reply:
.concat is the convention to add middleware in redux. But eslint have error because **prefer spread**